fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindRelation.java

```diff
@@ -460,8 +478,8 @@ private LogicalPlan getLogicalPlan(TableIf table, UnboundRelation unboundRelatio
         }
     }
 
-    private Plan parseAndAnalyzeHiveView(
-            HMSExternalTable table, String hiveCatalog, String hiveDb, String ddlSql, CascadesContext cascadesContext) {
+    private Plan parseAndAnalyzeExternalView(
```

Comment:
Here we should consider the dialect info in iceberg view's definition and dialect info user set in session variable.
If they are different, need to do something to handle it, like report an error.
For example, if the view's dialect is spark, and user specified session dialect is presto, then it will confuse the sql convertor about which dialect should it use to convert the view sql.